src/fidasim.f90

```diff
@@ -2811,7 +2921,7 @@ subroutine read_f(fid, error)
     endif
 
     if(inputs%verbose.ge.1) then
-        write(*,'(T2,"Distribution type: ",a)') "Fast-ion Density Function F(energy,pitch,R,Z)"
+        write(*,'(T2,"Distribution type: ",a)') "Fast-ion Density Function F(energy,pitch,R,Z,Phi)"
         write(*,'(T2,"Nenergy = ",i3)') fbm%nenergy
         write(*,'(T2,"Npitch  = ",i3)') fbm%npitch
```

Comment:
We should add the sizes of `nr`,`nz`, and `nphi` since the distributions `nphi` could be different.